alethzero/plugins/transactions/InjectTransactions.cpp

```diff
@@ -72,7 +72,7 @@ void InjectTransactions::doInject(QString _txHex)
 		bytes b = fromHex(_txHex.toStdString(), WhenError::Throw);
 		aleth()->ethereum()->injectTransaction(b);
 	}
-	catch (BadHexCharacter& _e)
+	catch (BadHexCharacter& /*_e*/)
```

Comment:
Would also prefer to remove it completely, but merging for now to enable building without warnings.